extend-vnet-to-multi-vnet/azuredeploy.json

```diff
@@ -0,0 +1,284 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+
+    "parameters": {
+
+        "existingVnetName": {
+            "type": "string",
+            "defaultValue": "vnet01",
+            "metadata": {
+                "description": "Name of the existing first VNET. This VNET must already have a subnet named GatewaySubnet defined with a minimum /29 address prefix"
+            }
+        },
+
+        "existingVnetLocation": {
+            "type": "string",
+            "allowedValues": [
```

Comment:
Looks like this list needs to include all Azure locations. Better idea might be to remove the allowed values and let user enter one.